packages/date-time/src/components/Calendar/CalendarYear/CalendarYear.styles.ts

```diff
@@ -0,0 +1,151 @@
+import { ICalendarYearStyleProps, ICalendarYearStyles } from './CalendarYear.types';
```

Comment:
I see what you meant about the styles. Right now this is a copy/paste of the existing CalendarMonth.styles.ts, but with different types and classNames so you can't necessarily just re-use as is. I'm all for a small refactoring that would let us eliminate this file if possible

Reply:
A minimally invasive refactor could be a common interface just for the shared styles, like ICalendarPickerStyles/StyleProps or something. Both pickers can base their own styles on it (or extend it). Make the props more generic:
- highlightCurrentMonth/Year -> highlightCurrent
- year/monthNavigationButton -> navigationButton
- etc...